packages/next/src/server/lib/trace/tracer.ts

```diff
@@ -34,11 +34,17 @@ const isPromise = <T>(p: any): p is Promise<T> => {
   return p !== null && typeof p === 'object' && typeof p.then === 'function'
 }
 
+type BubbledError = Error & { bubble?: boolean }
+
 const closeSpanWithError = (span: Span, error?: Error) => {
-  if (error) {
-    span.recordException(error)
+  if ((error as BubbledError | undefined)?.bubble === true) {
+    span.recordException('next.bubble')
```

Comment:
Should these be ignored completely or is `recordException` required here? The `bubble` field is being used internally by Next.js and could be changed to something more clear to signal it should be ignored if that helps

Reply:
Without failing the whole span, this record is simply an event - it's not super needed, but it also doesn't hurt.

Reply:
I went ahead and replace an error event with a tag. PTAL.